code/zoltar_scripts/upload_zoltar.py

```diff
@@ -0,0 +1,161 @@
+from __future__ import absolute_import
+
+# from zoltpy.quantile_io import json_io_dict_from_quantile_csv_file
+from zoltpy import util
+from zoltpy.connection import ZoltarConnection
+# from zoltpy.covid19 import COVID_TARGETS, COVID_ADDL_REQ_COLS, covid19_row_validator, validate_quantile_csv_file
+import os
+import sys
+
+import glob
+from pathlib import Path
+import pprint
+import yaml
+import logging
+import pickle
+import hashlib
+import json
+
+sys.path.append("validation/codebase")
+from quantile_io import json_io_dict_from_quantile_csv_file, REQUIRED_COLUMNS
+from covid19 import VALID_TARGET_NAMES, codes, covid19_row_validator, validate_quantile_csv_file
+from cdc_io import YYYY_MM_DD_DATE_FORMAT
+
+# logging.basicConfig(level=logging.DEBUG)
+
+cwd_p = Path(__file__).parent.resolve()
+all_forecasts = glob.glob('./data-processed/**/*-*.csv')
+pprint.pprint(all_forecasts)
+# meta info
+project_name = 'ECDC European COVID-19 Forecast Hub'
+project_obj = None
+conn = util.authenticate()
+url = 'https://github.com/epiforecasts/covid19-forecast-hub-europe/tree/main/data-processed'
+
+# all_forecasts = glob.glob('./data-processed')
```

Comment:
remove?

Reply:
yes, sorry, these are leftovers from adapting from US hub / Joe's code, thanks for flagging.